README.md

```diff
@@ -74,26 +74,57 @@ data and restart them (if they are running).
 fluo-dev reset all
 ```
 
-Verify that your `fluo.properties` file generated by `fluo configure` is correct
+Confirm that everything started by checking the monitoring pages of Hadoop & Accumulo:
+ * [Hadoop NameNode](http://localhost:50070/)
+ * [Hadoop ResourceManager](http://localhost:8088/)
+ * [Accumulo Monitor](http://localhost:50095/)
+
+Running Fluo
+------------
+
+With its dependencies running, Fluo can now be started.  Before starting Fluo, you can optionally
+verify or change configuration in the `fluo.properties` file generated by `fluo configure`.  
 ```
 vim conf/fluo/fluo.properties
 ```
 
-Finally, deploy Fluo which will remove any existing install, rebuild fluo, install fluo, and 
-configure it using `conf/fluo/fluo.properties`.
+If you want to run Fluo with observers, avoid configuring them in `fluo.properites`.  Instead, you
+should create a file called `observer.props` in `conf/` by copying the example:
+```
+cp conf/observer.props.example conf/observer.props
+vim conf/observer.props
+```
+
+The example `observer.props` file is configured to run the [fluo-stress][stress] example application.  
+The observer jar for [fluo-stress][stress] can be obtained by cloning and building its repo using the
+the steps below:
+```
+git clone https://github.com/fluo-io/fluo-stress.git
+cd fluo-stress/
+mvn package
+ls target/fluo-stress-*.jar
+```
+
+Copy your observer JAR for your application to `conf/fluo/observers`:
```

Comment:
A sentence explaining what to do if you later update the observer jar would be nice.